src/main/java/be/seeseemelk/mockbukkit/plugin/PluginManagerMock.java

```diff
@@ -459,12 +483,16 @@ private void addSection(PluginCommand command, String name, Object value)
 			command.setDescription((String) value);
 			break;
 		case "aliases":
-			List<String> aliases = new ArrayList<>();
 			if (value instanceof List<?>)
 				command.setAliases(
-				    ((List<?>) aliases).stream().map(Object::toString).collect(Collectors.toList()));
+						((List<?>)value).stream().map(Object::toString).collect(Collectors.toList()));
 			else
-				command.setAliases(Arrays.asList(value.toString()));
+				if(value != null)
+				{
+					command.setAliases(Collections.singletonList(value.toString()));
+				} else {
+					command.setAliases(Collections.emptyList());
+				}
```

Comment:
This if-statement is formatted really weirdly

Reply:
Fixed.